### PR TITLE
docs: Update install docs to Miniforge

### PR DESCRIPTION
**Pull request by lorenzncode**
I see that #1672 moved to Miniforge and that Mambaforge is deprecated (Miniforge itself now includes mamba).

I also added a PowerShell example.

### CODE REVIEW

File: cadquery/occ_impl/shapes.py [hunk 1]
```diff
@@ -2558,7 +2558,6 @@ def fillet(
         """
         Apply 2D or 3D fillet to a wire
 
-        :param wire: The input wire to fillet. Currently only open wires are supported
         :param radius: the radius of the fillet, must be > zero
         :param vertices: Optional list of vertices to fillet. By default all vertices are fillet.
```
Comment (nobkd) [inline cadquery/occ_impl/shapes.py:2562]:
```suggestion
        :param vertices: Optional list of vertices to fillet. By default all vertices are filleted.
```

Comment (lorenzncode) [reply cadquery/occ_impl/shapes.py:2562]: Thanks @nobkd .  I find the OCCT user guide phrases this as specifying vertices to be deleted.   In the context of creating a fillet on a planar face: 

"To create a fillet on planar face: define it, indicate, which vertex is to be deleted, and give the fillet radius ..."

I'll try an update along those lines.

**Review by adam-urbanczyk — APPROVED** [review]
LGTM!

**Comment by lorenzncode** [comment]
I found one more Miniforge link. I think that should be it.

**Review by jmwright — APPROVED** [review]
Looks good, thanks @lorenzncode

**Comment by adam-urbanczyk** [comment]
Alright, merging. Thanks a lot @lorenzncode !

**Pull request merged**